--- app/index.js	2024-08-19 11:55:41.166977147 +0200
+++ app/index.js	2024-08-19 11:56:35.043093758 +0200
@@ -89,6 +89,8 @@
   }
   mainScreen = require('./mainScreen');
   mainScreen.init();
+  // Bouquet injection
+  require('../bouquet').init();
   const {
     getWindow: getPopoutWindowByKey,
     getAllWindows: getAllPopoutWindows,

--- app/mainScreenPreload.js	2024-08-19 11:55:41.166977147 +0200
+++ app/mainScreenPreload.js	2024-08-19 11:57:07.269169516 +0200
@@ -65,6 +65,8 @@
       }
     });
   }
+  // Bouquet preload injection
+  require('../bouquet').preload();
   contextBridge.exposeInMainWorld('DiscordNative', DiscordNative);
   process.once('loaded', () => {
     global.DiscordNative = DiscordNative;
